### PR TITLE
P11SAK: Add option to list-key command to show MKVPs 

### PR DESCRIPTION
**Pull request by ifranzki**

### CODE REVIEW

File: usr/sbin/p11sak/p11sak.h [hunk 1]
```diff
@@ -322,10 +324,31 @@ enum p11sak_toke_type {
     TOKTYPE_EP11 = 2,
 };
 
+#define EP11_WKVP_OFFSET        32
+
+struct cca_token_header {
+    unsigned char id;
+    unsigned char reserver1;
```
Comment (jschmidb) [inline usr/sbin/p11sak/p11sak.h:331]: reserved

File: usr/sbin/p11sak/p11sak.h [hunk 2]
```diff
@@ -322,10 +324,31 @@ enum p11sak_toke_type {
...
+    unsigned short len;
+    unsigned char version;
+    unsigned char reserverd[3];
```
Comment (jschmidb) [inline usr/sbin/p11sak/p11sak.h:334]: reserved

File: man/man1/p11sak.1.in [hunk 3]
```diff
@@ -683,6 +684,11 @@ option. When option
 .BR \-\-long | \-l
 is specified, the keys are listed in long format, displaying the values of all
 attributes defined for the key type, including non-boolean attributes.
+When option
+.BR \-\-hsm\-mkvp | \-m
+is specified, the HSM master key verification patterns (MKVPs) of the keys are
+displayed. This is only possible for secure key tokens, such as the CCA and EP11
+tokens.
```
Comment (jschmidb) [inline man/man1/p11sak.1.in:691]: We could add a hint that this is also only possible/applicable for private keys. Same below ... 
Displaying public keys with --long and --hsm-mkvp just silently ignores the hsm parm.

Comment (jschmidb) [reply man/man1/p11sak.1.in:691]: Ah yes, and we should add a hint that the --hsm-mkvp option is only applicable together with --long

Comment (ifranzki) [reply man/man1/p11sak.1.in:691]: No, both is not true.  

In EP11 a public key is a MACed SPKI which is also bound to the EP11 master key. For CCA, public keys are not bound to the master keys though.

And no, `--hsm-mkvp` is NOT only available with --long, it works also without `--long`. Actually I think the way more useful use case is without `--long`.

Comment (jschmidb) [reply man/man1/p11sak.1.in:691]: OK, yes, I see it now, forget the two comments.

**Review by jschmidb — APPROVED** [review]
lgtm.

**Pull request merged**